### PR TITLE
KOGITO-7526: [yard Editor] First draft of yard UI editor

### PR DESCRIPTION
**Pull request by yesamer**
This is a draft proposal of a possible implementation of the `yard` editor UI.

**Web view**
Screenshot from 2022-07-12 16-14-58

**VSCode extension view**
Screenshot from 2022-07-11 16-23-41

This PR provides:
- In the Editor UI, a tab element is related to every section of a `yard` file (`General`, `Decision Elements` and `Decision Inputs`). Currently, only `Decision Elements` tab is enabled
- `Decision Elements` tab scope is to show the `boxed-expression-editor` related to the selected element (`yard` file may have multiple of them. Currently, an empty state is always shown. An example of future implementation is shown in the below screenshot: (SPOILER ALERT :D )
Screenshot from 2022-07-07 10-51-46
- `i18n` management
- `css` management

### CODE REVIEW

File: packages/yard-editor/src/i18n/locales/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
```
Comment (fantonangeli) [inline packages/yard-editor/src/i18n/locales/index.ts]: Copyright 2020

**Review by fantonangeli — APPROVED** [review]
Great code! Thank you @yesamer

**Review by caponetto — APPROVED** [review]
LGTM

**Comment by domhanak** [comment]
I can take a look tomorrow, if it is urgent I will review after merge. Looks great btw. @yesamer

**Comment by yesamer** [comment]
@domhanak I can wait, thank you!

**Review by domhanak — APPROVED** [review]
Thanks, looks good, we should plan a meeting to discuss and eventually merge the test helpers for testing of `vscode-*` packages into common package. @yesamer So I will be reaching out to you soon :)

**Comment by yesamer** [comment]
> Thanks, looks good, we should plan a meeting to discuss and eventually merge the test helpers for testing of `vscode-*` packages into common package. @yesamer So I will be reaching out to you soon :)

Great, thank you @domhanak !

**Pull request merged**